docs/source/tutorials/one_class_segmentation_2d_unet.rst

```diff
@@ -24,137 +27,345 @@ Download dataset
+
+
+    - ``command``: Action to perform. Here, we want to **train** a model:
+
+        .. tabs::
+
+            .. tab:: JSON File
+
+                We can set the field within the newly copied ``config.json`` file as follow, around `this line: <https://github.com/ivadomed/ivadomed/blob/master/ivadomed/config/config.json#L2>`__:
+
+                .. code-block:: json
+
+                    "command": "train"
+
+
+            .. tab:: Command Line Interface
+
+                Note that you can also pass this argument via CLI (see `Usage <../usage.html>`__)
+
+                .. code-block:: bash
+
+                        ivadomed --train -c path/to/config
 
-    In this tutorial we will use the configuration file: ``ivadomed/config/config.json``.
-    First off, copy this configuration file in your local directory (to avoid modifying the source file):
 
-    .. code-block:: bash
+    - ``path_output``: Folder name that will contain the output files (e.g., trained model, predictions, results).
 
-       cp <PATH_TO_IVADOMED>/ivadomed/config/config.json .
+        .. tabs::
 
-    Then, open it with a text editor. Below we will discuss some of the key parameters to perform a one-class 2D
-    segmentation training.
+            .. tab:: JSON File
 
-    - ``command``: Action to perform. Here, we want to train a model, so we set the fields as follows:
+                Usually around `this line: <https://github.com/ivadomed/ivadomed/blob/master/ivadomed/config/config.json#L4>`__ in the ``config.json`` is where you can update the ``path_output``.
 
-      .. code-block:: json
+                .. code-block:: json
 
-         "command": "train"
+                    "path_output": "spineGeneric"
 
-    Note that you can also pass this argument via CLI (see `Usage <../usage.html>`__)
+            .. tab:: Command Line Interface
 
-      .. code-block:: bash
+                Note that you can also pass this argument via CLI (see `Usage <../usage.html>`__)
 
-        ivadomed --train -c path/to/config
+                .. code-block:: bash
 
-    - ``path_output``: Folder name that will contain the output files (e.g., trained model, predictions, results).
+                    ivadomed -c path/to/config --path-output path/to/output/directory
 
-      .. code-block:: json
+    - ``loader_parameters:path_data``: Location of the dataset. As discussed in `Data <../data.html>`__, the dataset
+      should conform to the BIDS standard. Modify the path so it points to the location of the downloaded dataset.
 
-         "path_output": "spineGeneric"
+        .. tabs::
 
-    Note that you can also pass this argument via CLI (see `Usage <../usage.html>`__)
+            .. tab:: JSON File
 
-      .. code-block:: bash
+                Usually around **Line Number 12** in the ``config.json`` is where you can update the ``path_data`` within the ``loader_parameters`` sub-dictionary.
```

Comment:
There should be a link to the file instead of a specific line.

Reply:
Good eye! I think missed this link somehow.